docs/blog/2024-11-08-chicory-1.0.0-M1.md

```diff
@@ -285,8 +285,8 @@ calling a native executable.
           <configuration>
             <!-- Translate the Wasm binary `wat2wasm` into bytecode -->
             <wasmFile>src/main/resources/wat2wasm</wasmFile>
-            <!-- Generate the following class file as a result -->
```

Comment:
We should add a documentation section in the docs instead of changing the content of the blog.

Reply:
Agreed, I forgot that this blog post is about the published `-M1` artifact which won't be affected by this change.